### ISSUE TITLE
Create navigation bar component

### ISSUE THREAD

**Issue by VirginiaBalseiro** [Junior-Devs/2020-Junior-Devs-Project#23]
### Create a component for the navigation bar: 

![image](https://user-images.githubusercontent.com/28412960/80809252-9603c580-8bc1-11ea-8a69-52732e198ee7.png)

### Requirements: 

- [ ] Create the component
- [ ] Use `#` as a placeholder for links for now, there will be another ticket to update this.
- [ ] Write tests - we'll use https://github.com/testing-library/react-testing-library
- [ ] Link to design where you can find the CSS: https://www.figma.com/file/HNLdpIu2S7kV0G03By3Sh1/Elearning-Platform-(home)?node-id=1%3A3

**Comment by akdsco** [comment]
I can work on this too :) Should I assume no mobile/tablet optimisation or improvise?

**Comment by VirginiaBalseiro** [comment]
@akdsco You were assigned to another ticket, and we'll allowing people to work on one ticket at a time, so I'll keep this one unassigned for someone else to work on it while you work on the other one. Thanks.

**Comment by Luma1970** [comment]
Hi, I would work on this. Two question: Should I create this component in React? Do the "Register" and "Login" features should be active or do you want only the HTML and CSS component?

**Comment by VirginiaBalseiro** [comment]
@Luma1970 Yes, please! You can use the templates or other components as examples. The buttons should be just placeholders for now until we have the login and register pages.

**Comment by Luma1970** [comment]
> @Luma1970 Yes, please! You can use the templates or other components as examples. The buttons should be just placeholders for now until we have the login and register pages.

Perfect! So if you assigned it to me I start to work on it :)

**Comment by VirginiaBalseiro** [comment]
@Luma1970 assigned :)
